provisioner/powershell/provisioner.go

```diff
@@ -112,19 +113,19 @@ func (p *Provisioner) Prepare(raws ...interface{}) error {
 	}
 
 	if p.config.EnvVarFormat == "" {
-		p.config.EnvVarFormat = `$env:%s="%s"; `
+		p.config.EnvVarFormat = `$env:%s=\"%s\"; `
```

Comment:
did you intend to escape these but not the elevated env vars?

Reply:
Yes - that's right. This is due to the difference in how we now incorporate env vars in normal -vs- elevated PowerShell commands.

For the normal command, the env vars are incorporated directly in the command string. _Since the command string itself makes use of double quotes_, we need to escape the quotes surrounding the env var values so we end up with a command something like:

```
powershell -executionpolicy bypass "& { if (Test-Path ... blah};$env:FOO=\"bar\"; $env:BAR=\"baz\";...;  &'c:/Windows/Temp/script-59ca48a4-f4d6-f919-4653-6b3a95e3bb48.ps1';exit $LastExitCode }"
```

We don't need to escape the double quotes surrounding the env var values for elevated commands. Uploading the env vars into a file and dot sourcing them means the env vars don't appear directly in the elevated command string and so we don't need to escape the double quotes.

Hope I've made sense there...